subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/JavaCompileSpec.java

```diff
@@ -39,17 +58,17 @@ public interface JavaCompileSpec extends JvmLanguageCompileSpec {
 
     Set<AnnotationProcessorDeclaration> getEffectiveAnnotationProcessors();
 
-    void setClasses(Set<String> classes);
+    void setClassesToProcess(Set<String> classes);
```

Comment:
❌ I think these methods could use some Javadocs. What is the difference between the classes to process and the classes to compile?

Reply:
Added some documentation